cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb

```diff
@@ -18,9 +18,51 @@
 node.default['cluster']['scripts_dir'] = '/opt/parallelcluster/scripts'
 node.default['cluster']['region'] = node['region']
 node.default['cluster']['slurmdbd_response_retries'] = 30
-# TODO: attributes for stack name
+node.default['cluster']['stack_name'] = node['stack_name']
+node.default['cluster']['munge']['user'] = 'munge'
+node.default['cluster']['munge']['group'] = node['cluster']['munge']['user']
+
+if platform?('amazon') && node['platform_version'] == "2"
+  node.default['cluster']['cluster_user'] = 'ec2-user'
```

Comment:
Maybe we should use the `users_amazon2.rb` file

Reply:
I deleted the code. It is not needed